wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -262,7 +262,8 @@ def get_searchable_content(self, value):
 
 class RichTextBlock(FieldBlock):
 
-    def __init__(self, required=True, help_text=None, **kwargs):
+    def __init__(self, required=True, help_text=None, widget=None, **kwargs):
+        self.widget = widget
```

Comment:
I agree with your first proposed approach, this pattern is used mostly throughout other specific block types and upholding this consistency makes most sense to me.

As to your proposed solution, I'd prefer using the standard get method of the dictionary. This better covers all cases of a non existing key as well as an existing key returning None. The naming conventions in the below code now also follows used patterns.

```python
def field(self):
    field_kwargs = self.field_options.copy()
    # check if a custom widget has been set or use default
    if not field_kwargs.get('widget'):
        from wagtail.wagtailcore.fields import RichTextArea
        field_kwargs.update({'widget': RichTextArea})
    return forms.CharField(**field_kwargs)

I've update the branch with this solution. I'm happy for your feedback if you see room for improvements.